service/sqs/service.go

```diff
@@ -8,7 +8,8 @@ import (
 	"github.com/aws/aws-sdk-go/aws/client/metadata"
 	"github.com/aws/aws-sdk-go/aws/request"
 	"github.com/aws/aws-sdk-go/aws/signer/v4"
-	"github.com/aws/aws-sdk-go/private/protocol/query"
+	"github.com/aws/aws-sdk-go/private/protocol"
+	"github.com/aws/aws-sdk-go/private/protocol/jsonrpc"
 )
```

Comment:
Can we add sqs specific unit tests? Mock raw json response and make sure a valid error code is returned.

Reply:
Sure. Will add some unit tests to validate the error codes.

Reply:
I see that you have added examples with the integration tests. How do we know if they are run? What target should be invoked to run them as integration tests? And, why do they have "mockClients" in their path? What exactly is being mocked?